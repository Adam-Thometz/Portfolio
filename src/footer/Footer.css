.Footer {
  display: flex;
  justify-content: space-evenly;
  animation: slide-up 0.5s 1.2s both;
}

.Footer-link {
  text-decoration: none;
  color: rgb(207, 207, 207);
  margin-left: 15px;
}