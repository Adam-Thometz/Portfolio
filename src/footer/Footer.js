import React from 'react'

import './Footer.css';

import Links from './links/Links';

const Footer = () => {
  return (
    <footer className='Footer'>
      <Links />
      <small>Made by Adam Thometz 2022</small>
    </footer>
  );
};

export default Footer;