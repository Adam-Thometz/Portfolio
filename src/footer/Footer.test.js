import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

describe("Footer component", () => {
  it("renders without crashing", () => {
    render(<Footer />);
  });

  it("matches the snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
