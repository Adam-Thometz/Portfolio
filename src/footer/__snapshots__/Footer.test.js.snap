// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches the snapshot 1`] = `
<DocumentFragment>
  <footer
    class="Footer"
  >
    <small>
      Made by Adam Thometz 2022
    </small>
    <small>
      <a
        class="Footer-link"
        href="https://github.com/Adam-Thometz/Portfolio"
        rel="noreferrer"
        target="_blank"
      >
        Like this portfolio? Click here to check out the Github
      </a>
    </small>
  </footer>
</DocumentFragment>
`;
